app/controllers/v1/audits_controller.rb

```diff
@@ -17,16 +17,11 @@ def show
 
     def create
       collection_policy.new(current_user).authorize! :new?
-
-      packages = Package.stored
-      audit = Audit.new(user: current_user, packages: packages.count)
+      audit = Audit.new(user: current_user, packages: Package.stored.count)
```

Comment:
Because we're not saving in this method anymore, we don't have to set the package count. Just give it what we know here, but the audit method cannot know -- the user.